dll/win32/shell32/dialogs/filetypes.cpp

```diff
@@ -660,12 +660,11 @@ FileTypesDlg_InsertToLV(HWND hListView, LPCWSTR szName, INT iItem, LPCWSTR szFil
     }
 
     // add icon to imagelist
-    INT iLargeImage = -1, iSmallImage = -1;
+    INT iSmallImage = -1;
     if (Entry->hIconLarge && Entry->hIconSmall)
     {
-        iLargeImage = ImageList_AddIcon(himlLarge, Entry->hIconLarge);
+        ImageList_AddIcon(himlLarge, Entry->hIconLarge);
         iSmallImage = ImageList_AddIcon(himlSmall, Entry->hIconSmall);
-        ASSERT(iLargeImage == iSmallImage);
```

Comment:
You could use `DBG_UNREFERENCED_LOCAL_VARIABLE` instead

Reply:
I do confirm that DBG_UNREFERENCED_LOCAL_VARIABLE(iLargeImage);
does work as well, and is accepted by the followjng compilers without any warnings (even in old releases/0.4.10-branch):
MSVC2010SP1 dbg-cfg
MSVC2010SP1 rls-cfg
GCC4.7.2 dbg-cfg
GCC4.7.2 rls-cfg

I do agree, that this is at least slightly less ugly than the variant with the ifdefs that I had in mind as an alternative.
Ok, I will do that.